tasks1.js

```diff
@@ -0,0 +1,119 @@
+// 1
+function convertMinutesToHours(minutes) {
+  return (Math.floor(minutes / 60)) + ':' + (minutes % 60);
+}
+console.log(convertMinutesToHours(75));
+
+function convertMinutesToHoursInNumber(minutes) {
+  return (Math.floor(minutes / 60)) + (minutes % 60) / 100;
+}
+console.log(convertMinutesToHoursInNumber(75));
+
+
+// 2
+function isTextUppercase(text) {
+  return text.toUpperCase() === text;
+}
+console.log(isTextUppercase('START'));
+
+
+// 3
+function isDivisible(dividedNumber, divideBy1, divideBy2) {
```

Comment:
Those names still need improvement.
`dividedNumber` - this number is not divided yet. A better name would be `numberToDivide`.
Using numbers in the variable and argument names makes them more difficult to read. I suggest `firstNumberToDivideBy` and `secondNumberToDivideBy`.